app/_locales/en/messages.json

```diff
@@ -1937,7 +1937,12 @@
     "message": "Using the best quote"
   },
   "swapVerifyTokenExplanation": {
-    "message": "Multiple tokens can use the same name and symbol. Check Etherscan to verify this is the token you're looking for."
+    "message": "Multiple tokens can use the same name and symbol. Check Etherscan to verify this is the token you're looking for.",
+    "description": "This appears in a tooltip next to the verifyThisTokenOn message. It gives the user more information about why they should check the token on a block explorer. This is the message that appears when we want to direct the user to etherscan."
+  },
+  "swapVerifyTokenOnBlockExplorerExplanation": {
+    "message": "Multiple tokens can use the same name and symbol. Check $1 to verify this is the token you're looking for.",
```

Comment:
Hmm. Could we take this same approach with each other locale too by substituting `Etherscan` for `$1`, which would be `Etherscan` when Etherscan was the block explorer being used? That would ensure we have _something_ understandable for this message in each locale at least, and we know that the sentences are all gramatically correct with proper noun where 'Etherscan' is.

I guess it's possible something about the sentence could change depending on the input, but.... I don't know, that seems difficult to handle perfectly for all cases even if we wanted to.

If we consolidate these two messages, we can also simplify the logic used in `build-quote.js`, because the tooltip text would be the same for each branch.

Reply:
Yeah, I think this is a good idea. Done in 1cfbe6c5e